packages/angular/cli/utilities/check-package-manager.ts

```diff
@@ -1,53 +0,0 @@
-/**
```

Comment:
why is this file deleted? is it an unrelated cleanup?

Reply:
Yes. It’s a separate commit.  I can move it out if you want.